lib/utils/device/deploy.ts

```diff
@@ -121,6 +126,58 @@ async function environmentFromInput(
 	return ret;
 }
 
+async function buildArgumentsFromInput(
+	buildArgs: string[],
+	serviceNames: string[],
+	logger: Logger,
+): Promise<ParsedBuildArguments> {
+	const { exitWithExpectedError } = await import('../../errors');
```

Comment:
`exitWithExpectedError`  is deprecated, in favour of throwing an `ExpectedError`:

```
	const { ExpectedError } = await import('../../errors');
        throw new ExpectedError('Some error');
```

This is then caught and handled by the error handler, rather than exiting ungracefully, which might cause problems.